src/store/entities/Product.java

```diff
@@ -1,21 +1,24 @@
 package store.entities;
 
-public class Product {
+import java.io.Serializable;
 
+public class Product implements Serializable {
+
+	private static final long serialVersionUID = 1L;
 	private String name;
 	private double currentPrice;
 	private int stockOnHand;
 	private int reOrderLevel;
 	private String id;
+	private boolean isOrdered;
 
-	private static int idCounter = 1;
-
-	public Product(String name, double currentPrice, int stockOnHand, int reOrderLevel) {
+	public Product(String name, String id, double currentPrice, int stockOnHand, int reOrderLevel) {
 		this.name = name;
 		this.currentPrice = currentPrice;
 		this.stockOnHand = stockOnHand;
 		this.reOrderLevel = reOrderLevel;
-		this.id = "P-" + idCounter++;
```

Comment:
Could we keep this in? I thought it made sense to have member id's be M-# and product id's be P-#